src/Rector/Contrib/PHPUnit/SpecificMethod/AssertCompareToSpecificMethodRector.php

```diff
@@ -35,27 +36,41 @@
         'gettype' => ['assertInternalType', 'assertNotInternalType'],
     ];
 
+    /**
+     * @var string[][]|string[]
+     */
+    private $acceptedAssertionMethods = [
+        'trueMethodNames' => ['assertSame', 'assertEquals'],
+        'falseMethodNames' => ['assertNotSame', 'assertNotEquals'],
+    ];
+
     /**
      * @var MethodCallAnalyzer
      */
     private $methodCallAnalyzer;
 
+    /**
+     * @var MethodNameChanger
+     */
+    private $methodNameChanger;
+
     /**
      * @var string|null
      */
     private $activeFuncCallName;
 
-    public function __construct(MethodCallAnalyzer $methodCallAnalyzer)
+    public function __construct(MethodCallAnalyzer $methodCallAnalyzer, MethodNameChanger $methodNameChanger)
     {
         $this->methodCallAnalyzer = $methodCallAnalyzer;
+        $this->methodNameChanger = $methodNameChanger;
     }
 
     public function isCandidate(Node $node): bool
     {
         if (! $this->methodCallAnalyzer->isTypesAndMethods(
             $node,
             ['PHPUnit\Framework\TestCase', 'PHPUnit_Framework_TestCase'],
-            ['assertSame', 'assertNotSame', 'assertEquals', 'assertNotEquals']
+            Arrays::flatten($this->acceptedAssertionMethods)
```

Comment:
@TomasVotruba I used `Arrays::flatten` to avoid duplicate the `$acceptedAssertionMethods` already defined.